### PR TITLE
Add HeaderHelper

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | -

- Add `HeaderHelper::contentDispositionValue` that generate `Content-Disposition` header value
- Update dev dependencies
- Improve readme and changelog

### CODE REVIEW

File: src/HeaderHelper.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+     * - Could be omitted if it exactly matches fallback name.
+     *
+     * @see https://tools.ietf.org/html/rfc6266#page-5
+     *
+     * @param string $disposition
+     * @param string $attachmentName
+     *
+     * @return string
+     */
+    public static function contentDispositionValue(string $disposition, string $attachmentName): string
```
Comment (roxblnfk) [inline src/HeaderHelper.php]: `$attachmentName` shoul be optional

File: tests/HeaderHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            [
+                'attachment',
+                "a\x7Fb.png",
+                'attachment; filename="a_b.png"; filename*=utf-8\'\'a%7Fb.png',
+            ],
+            [
+                'attachment',
+                'a%20b.png',
+                'attachment; filename="a_20b.png"; filename*=utf-8\'\'a20b.png',
+            ],
```
Comment (roxblnfk) [inline tests/HeaderHelperTest.php]: could you add test cases for quotes, EOL chars, some unicode symbols (likre ё, ü, ö, ä, ß) and emoji?

File: .github/workflows/bc.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+on:
```
Comment (samdark) [inline .github/workflows/bc.yml:1]: Name it as `_bc.yml` for now.

**Review by roxblnfk — CHANGES_REQUESTED** [review]

File: src/HeaderHelper.php (hunk 1 above)
Comment (samdark) [inline src/HeaderHelper.php]: Do you have any other method candidates for this class?

File: .github/workflows/bc.yml (hunk 3 above)
Comment (vjik) [reply .github/workflows/bc.yml:1]: Why?
yiisoft/http released

File: src/HeaderHelper.php (hunk 1 above)
Comment (vjik) [reply src/HeaderHelper.php]: Not know

Comment (vjik) [reply src/HeaderHelper.php]: Done

File: tests/HeaderHelperTest.php (hunk 2 above)
Comment (vjik) [reply tests/HeaderHelperTest.php]: Add all, except EOL.
What is better: EOL replace to space or underscope?

**Comment by samdark** [comment]
It may be a bit more complicated: http://test.greenbytes.de/tech/tc2231/#attwithfilenamepctandiso

File: tests/HeaderHelperTest.php (hunk 2 above)
Comment (samdark) [reply tests/HeaderHelperTest.php]: Underscore.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍
